05-html-css/the-web.md

```diff
@@ -1,77 +1,101 @@
+</details>
+
+## How the Internet works: A more technical look
+
+Should you be excited to look at how the Internet sends and receives data, here are more resources for you.
+
+![How Information Travels on the Web](imgs/netdiag.gif)
+
+- [Analogy between postal mail and TCP/IP](http://bpastudio.csudh.edu/fac/lpress/471/hout/netech/postofficelayers.htm)
+- [Warriors of the Net Video](https://www.youtube.com/watch?v=PBWhzz_Gn10)
 
 ## Browser Compatibility
 
 The internet has changed a lot since Sir Tim Berners Lee made the first [web page](http://info.cern.ch/) in 1991. In turn, HTML and CSS have evolved from when they were first used in the 90's. As these front-end languages evolve, browsers need to evolve with them.
 
-The current versions of browsers we are using right now  understand HTML5 and CSS3, which utilize newest web development standards we will be learning.
+We will be learning HTML5 and CSS3. Be aware that some older browsers do not understand the newest versions of HTML or CSS. One day you many need to develop a site that needs to be accessed by a large demographic, including users with computers from 2001 (or older). Your site may break, or look a lot different on those computer's browsers.
 
-Be aware that some older browsers do not understand the newest versions of HTML or CSS. One day you many need to develop a site that needs to be accessed by a large demographic, including users with computers from 2001 (or older). Your site may break, or look a lot different on those computer's browsers.
+There are also many different browsers (Safari, Internet Explorer, Chrome, Firefox etc). A browser's job is to translate the code that will be displayed. Each browser does this a little bit differently. To ensure a site is widely accessible, developers use browser compatibility tools.
 
-There are also many different browsers (Safari, Internet Explorer, Chrome, Firefox etc). A browser's job is to translate the code that will be displayed. Each browser does this a little bit differently. To ensure a site is widely accessible, developers use browser [compatibility tools](http://www.catswhocode.com/blog/15-techniques-and-tools-for-cross-browser-css-coding).
+In the Ada curriculum, we will focus our learning goals on developing web sites that are compatible with the latest version of [Google Chrome](https://www.google.com/chrome/). Students don't need to switch to Google Chrome for their every day use, but are asked to use Google Chrome for web development at Ada.
 
-As we begin learning about HTML and CSS, we are not going to be concerned about our sites' browser compatibility. Instead we are only going to focus on developing with the most recent version of [Chrome](https://www.google.com/chrome/).  
+## 🔑 Key Takeaway
 
+There's a lot to web development and web!
 
-## Vocab ✅
-  - Web application
-  - Static
-  - Dynamic
-  - Front-end
-  - Back-end
-  - Client
-  - Server
```

Comment:
Mostly just felt that it wasn't as relevant to give a list of vocab, and I didn't find that they were no longer aligned with learning goals. I don't particularly care to emphasize front-end/back-end to them anymore (particularly because front-end is weird and we're teaching them full-stack, and students tend to accidentally pigeon-hole themselves into one bucket or the other when I find that's distracting most of the time).

Also, they've already learned client/server in a previous week at this point by now.